### ISSUE TITLE
Signature comment is always included in prqlc output

### ISSUE THREAD

**Issue by zhicwu** [PRQL/prql#2954]
### What happened?

prlqc command line has an option `--include-signature-comment` to explicitly include signature comment in output. However, the comment is included regardless using the option or not.

```bash
$ echo 'from t' | prqlc compile                                
SELECT
  *
FROM
  t

-- Generated by PRQL compiler version:0.8.1 (https://prql-lang.org)

$ echo 'from t' | prqlc compile --include-signature-comment
SELECT
  *
FROM
  t

-- Generated by PRQL compiler version:0.8.1 (https://prql-lang.org)
```

### PRQL input

```elm
from t
```

### SQL output

```SQL
SELECT
  *
FROM
  t

-- Generated by PRQL compiler version:0.8.1 (https://prql-lang.org)
```

### Expected SQL output

```SQL
SELECT
  *
FROM
  t
```

### MVCE confirmation

- [ ] Minimal example
- [ ] New issue

**Comment by eitsupi** [comment]
Could you try the latest dev version?

For example, I tried the latest nightly build of prqlc, there are not such a option, but there is the option `--hide-signature-comment`.
https://github.com/PRQL/prql/actions/runs/5463398902

```sh
❯ ./prqlc.exe compile --help
Parse, resolve, lower into RQ & compile to SQL

Only displays the main pipeline and does not handle loop.

Usage: prqlc.exe compile [OPTIONS] [INPUT] [OUTPUT] [MAIN_PATH]

Arguments:
  [INPUT]
          [default: -]

  [OUTPUT]
          [default: -]

  [MAIN_PATH]
          Identifier of the main pipeline

Options:
      --hide-signature-comment


  -t, --target <TARGET>
          [env: PRQLC_TARGET=]
          [default: sql.any]

      --color <WHEN>
          Controls when to use color

          [default: auto]
          [possible values: auto, always, never]

  -h, --help
          Print help (see a summary with '-h')
```

```sh
❯ echo 'from foo' | ./prqlc.exe compile --hide-signature-comment
Enter PRQL, then press ctrl-z to compile:

SELECT
  *
FROM
  foo

❯ echo 'from foo' | ./prqlc.exe compile --include-signature-comment
error: unexpected argument '--include-signature-comment' found

  tip: a similar argument exists: '--hide-signature-comment'

Usage: prqlc.exe compile --hide-signature-comment [INPUT] [OUTPUT] [MAIN_PATH]

For more information, try '--help'.
```

**Comment by aljazerzen** [comment]
Correct, this was fixed with #2649
